content/fr/security-chaos-engineering.md

```diff
@@ -0,0 +1,33 @@
+---
+title: La sécurité par l'ingénierie du chaos
+status: Completed
+category: concept
+tags: ["sécurité", "méthodologie", ""]
+---
+
+La sécurité par l'ingénierie du chaos, ou SCE (Security Chaos Engineering en anglais), est une discipline basée sur l'[ingénierie du chaos](/fr/chaos-engineering/).
+La SCE réalise des tests de sécurité proactifs sur un système distribué
+pour accroître la confiance dans la capacité du système à faire face à des conditions instables et malveillantes.
+Les ingénieurs en sécurité du chaos utilisent en boucles des méthodes scientifique pour y parvenir,
```

Comment:
problématique de traduction sur  :   "use **scientific method loops** to achieve this, including"...

- les italiens l'ont traduites en "utilisent des cycles de méthodes scientifiques pour atteindre cet objectif, notamment....", 
- les espagnols ont fait disparaitre le terme "loop",  -> "utilisent des méthodes scientifiques, notamment "
- les japonais : "utilisent une boucle de méthodes scientifiques qui comprend ...."

la version italienne est peut etre la meilleure ?
votre avis ?

Reply:
La traduction italienne me parait pas top, "des cycles de méthodes scientifiques" je ne suis pas sur que ça aie beaucoup de sens en français. 
Est-ce qu'on ne pourrais pas simplement déplacer des virgules pour rendre la lecture plus claire et insiuster sur le "en boucle"? 

Les ingénieurs en sécurité du chaos utilisent, en boucle, des méthodes scientifiques pour y parvenir comme la définition de l'état d'équilibre (steady-state), d'hypothèses, l'amélioration continue, l'apprentissage et la mise en œuvre de mesures d'atténuation.

Sinon ya aussi les locutions latines, mais ça fait pompeux: *ad infinitum* ou même *ad nauseam* a la place de "en boucle"

Reply:
je vais  mettre les virgules  : :p  le latin je ne suis pas fan.